template/nuxt/nuxt.config.js

```diff
@@ -82,7 +82,11 @@ module.exports = {
   /*
   ** Nuxt.js dev-modules
   */
+
   buildModules: [
+    <%_ if (typescript) {_%>
+    '@nuxt/typescript-build',
+    <%_ } _%>
     <%_ if (eslint) { _%>
     // Doc: https://github.com/nuxt-community/eslint-module
     '@nuxtjs/eslint-module',
```

Comment:
I think `@nuxtjs/eslint-module` can be replaced by `eslint` option for TypeScript checker : https://github.com/TypeStrong/fork-ts-checker-webpack-plugin#eslint